src/renderer/routes/test_sequencer_panel/utils/SequenceHandler.ts

```diff
@@ -229,6 +229,9 @@ async function saveToDisk(
 }
 
 async function installDeps(sequence: TestSequencerProject): Promise<boolean> {
+  if (sequence.interpreter.requirementsPath !== null) {
+    return true;
+  }
```

Comment:
shouldn't this be the other way around? i.e `=== null`

Reply:
Oh nice catch